Tasks/PerformActionOnMachineGroup/DtlHelper.ps1

```diff
@@ -105,3 +119,46 @@ function Block-MachineGroup
     Set-TaskVariable -Variable "DTL_RESERVATION_CONTEXT" -Value $blockedFor
     Write-Verbose "Task variable DTL_RESERVATION_CONTEXT set with the value $blockedFor"
 }
+
+function Invoke-WithRetry {
+    param(    
+    [Parameter(Mandatory)]$Command,
+    [Parameter(Mandatory)]$RetryDuration = 30,
+    [Parameter(Mandatory)]$OperationDetail,
+    $RetryDelay = 30)
```

Comment:
please add comment that it is in seconds because above retryduration is also 30 but that is in minutes